### PR TITLE
Update es.po (menu)

### PR DESCRIPTION
**Pull request by FrannDzs**
Spanish translation of the menu updated to date. 
It may contain some errors.

### CODE REVIEW

File: Source/Locales/Menu/es.po [hunk 1]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+#: ../../Menu/MainForm.Designer.cs:680
 msgid "Difficulty:"
-msgstr "Dificultad"
+msgstr "Dificultad:"
 
-#: ../../Menu/MainForm.Designer.cs:665 ../../Menu/MainForm.Designer.cs:788
+#: ../../Menu/MainForm.Designer.cs:702 ../../Menu/MainForm.Designer.cs:848
 msgid "Season:"
-msgstr "Estación meteorológica"
+msgstr "Temporada:"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:375]: Creo que es más adecuado mantener el término "Estación", puesto que se refiere a la estación del año en que ocurrirá la simulación.

File: Source/Locales/Menu/es.po [hunk 2]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
-msgstr ""
+msgstr "Estado de señal"
 
-#: ../../Menu/Options.cs:234
-#, fuzzy
-#| msgid "Super-elevation"
+#: ../../Menu/Options.cs:246
 msgid "Track Elevation"
-msgstr "Peralte"
+msgstr "Elevación de vía"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:496]: El término técnico es "peralte", por lo que optaría por mantener el término anterior

File: Source/Locales/Menu/es.po [hunk 3]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+#: ../../Menu/Options.cs:250
 msgid "Throttle %"
-msgstr ""
+msgstr "Aceleración %"
 
-#: ../../Menu/Options.cs:239
+#: ../../Menu/Options.cs:251
 msgid "Brake Cyl Press"
-msgstr ""
+msgstr "Prensa de cilindros de freno"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:516]:
```suggestion
msgstr "Presión de cilindros de freno"
```

File: Source/Locales/Menu/es.po [hunk 4]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
 "{0:F0}° vertical FOV is the same as:\n"
 "{1:F0}° horizontal FOV on 4:3\n"
 "{2:F0}° horizontal FOV on 16:9"
 msgstr ""
-"{0:F0}° de FOV vertical es lo mismo que:\n"
-"{1:F0}° de FOV horizontal en 4:3\n"
-"{2:F0}° de FOV horizontal en 16:9"
+"{0:F0}° vertical FOV is the same as:\n"
+"{1:F0}° horizontal FOV on 4:3\n"
+"{2:F0}° horizontal FOV on 16:9"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:624]: ¿Hay algún motivo para dejarlo en inglés?

File: Source/Locales/Menu/es.po [hunk 5]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+#: ../../Menu/Options.Designer.cs:274
 msgid "OK"
 msgstr "Aceptar"
 
-#: ../../Menu/Options.Designer.cs:272
-msgid "Brake pipe charging rate (PSI/s)"
-msgstr "Velocidad de carga de la tubería de freno (PSI/s)"
+#: ../../Menu/Options.Designer.cs:308
+msgid "Brake pipe charging rate (PSI/s)             "
+msgstr "Velocidad de carga del tubo de freno (PSI/s)"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:662]: En España se dice "tubería de freno". Si en otros países se dice "tubo" igual habría que hacer sublocalizaciones.

File: Source/Locales/Menu/es.po [hunk 6]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+msgstr ""
+
+#: ../../Menu/Options.Designer.cs:951
+msgid "Shadow for all shapes"
+msgstr "Sombra para todas las formas"
+
+#: ../../Menu/Options.Designer.cs:961
 msgid "Double overhead wires"
-msgstr "Doble hilo de catenaria"
+msgstr "Cables aéreos dobles"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:762]: Aquí también prefiero la traducción anterior, el término técnico es "catenaria".

File: Source/Locales/Menu/es.po [hunk 7]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
-#: ../../Menu/Options.Designer.cs:929
+#: ../../Menu/Options.Designer.cs:1256
 msgid "Dynamic shadows"
 msgstr "Sombras dinámicas"
 
-#: ../../Menu/Options.Designer.cs:939
+#: ../../Menu/Options.Designer.cs:1266
 msgid "Overhead wire"
-msgstr "Hilo de catenaria"
+msgstr "Cables aéreos"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:819]: Igual, el término correcto es "catenaria"

File: Source/Locales/Menu/es.po [hunk 8]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+#: ../../Menu/Options.Designer.cs:1289
 msgid "Simulation"
 msgstr "Simulación"
 
-#: ../../Menu/Options.Designer.cs:974
-#, fuzzy
-#| msgid "Activity Briefing"
+#: ../../Menu/Options.Designer.cs:1299
+msgid "Diesel engines stopped after startup"
+msgstr "Los motores diésel se paran tras el arranque"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:827]:
```suggestion
msgstr "Motores diésel parados al inicio"
```

File: Source/Locales/Menu/es.po [hunk 9]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+#: ../../Menu/Options.Designer.cs:1453
 msgid "Break couplers"
-msgstr "Rotura de enganches"
+msgstr "Romper acomplamientos"
 
-#: ../../Menu/Options.Designer.cs:1115
+#: ../../Menu/Options.Designer.cs:1463
 msgid "Advanced adhesion model"
-msgstr "Usar modelo de adhesión avanzada"
+msgstr "Modelo avanzado de adhesión"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:883]:
```suggestion
msgstr "Modelo avanzado de adherencia"
```

File: Source/Locales/Menu/es.po [hunk 10]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+#: ../../Menu/Options.Designer.cs:1696
 msgid "Interval (sec)"
-msgstr "Intervalo (Seg)"
+msgstr "Intervalo (seg)"
 
-#: ../../Menu/Options.Designer.cs:1346
+#: ../../Menu/Options.Designer.cs:1705
 msgid "Log station stops"
-msgstr "Registro de paradas en estaciones"
+msgstr "Registro de paradas de la estación"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:947]: Prefiero el término anterior "Registro de parada en estaciones"

File: Source/Locales/Menu/es.po [hunk 11]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+#: ../../Menu/Options.Designer.cs:2303
 msgid "Level:"
-msgstr "Nivel"
+msgstr "Nivel:"
 
-#: ../../Menu/Options.Designer.cs:1985
+#: ../../Menu/Options.Designer.cs:2343
 msgid "Adhesion proportional to rain/snow/fog"
-msgstr ""
+msgstr "Adhesión proporcional a la lluvia/nieve/niebla"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:1048]:
```suggestion
msgstr "Adherencia proporcional a la lluvia/nieve/niebla"
```

File: Source/Locales/Menu/es.po [hunk 12]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
 msgid "Extend object maximum viewing distance to horizon"
-msgstr ""
-"Ampliar la distancia máxima de visualización del objeto hasta el horizonte"
+msgstr "Ampliar la distancia máxima de visión del objeto hasta el horizonte"
 
-#: ../../Menu/Options.Designer.cs:2097
+#: ../../Menu/Options.Designer.cs:2445
 msgid "Gauge (mm)"
-msgstr "Galga (mm)"
+msgstr "Calibre (mm)"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:1077]: Normalmente se utiliza "Galga" o "Ancho de vía"

File: Source/Locales/Menu/es.po [hunk 13]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
-"Las funciones experimentales pueden disminuir el rendimiento del juego, "
-"utilicelas bajo su propio riesgo."
+"Funciones experimentales que pueden ralentizar el juego, utilízalas bajo tu "
+"propia responsabilidad."
 
-#: ../../Menu/Options.Designer.cs:2210
+#: ../../Menu/Options.Designer.cs:2558
 msgid "Super-elevation"
-msgstr "Peralte"
+msgstr "Superelevación"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:1092]:
```suggestion
msgstr "Peralte"
```

File: Source/Locales/Menu/es.po [hunk 14]
```diff
@@ -79,1297 +79,1311 @@ msgstr "Cancelar cambios"
...
+#: ../../Menu/Options.Designer.cs:2332
 msgid "Default is 130%"
-msgstr "El valor predeterminado es 100"
+msgstr "Por defecto es el 130%"
 
-#: ../../Menu/Options.Designer.cs:2159
+#: ../../Menu/Options.Designer.cs:2507
 msgid "Shortest curve to have elevation"
-msgstr "Radio mínimo en curva para peralte"
+msgstr "La curva más corta para tener elevación"
```
Comment (cesarBLG) [inline Source/Locales/Menu/es.po:1188]: Prefiero el término anterior

**Review by cesarBLG — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution! Most things seem OK, just there are some translations which were more accurate previously from a technical point of view.

**Comment by FrannDzs** [comment]
> 

Hi, nice to meet you. I had doubts about some terms, thanks for checking the commit. I don't have a deep knowledge of railways, so the help is much appreciated.

File: Source/Locales/Menu/es.po (hunk 4 above)
Comment (FrannDzs) [reply Source/Locales/Menu/es.po:624]: Recuerdo haberlo traducido no sé que pasó. (?

File: Source/Locales/Menu/es.po (hunk 6 above)
Comment (FrannDzs) [reply Source/Locales/Menu/es.po:762]: Concuerdo.

**Review by cesarBLG — APPROVED** [review]
I think it is good. For future contributions to github projects, I'd suggest creating separate branches for each feature, instead of making the changes in the "master" branch directly.

**Comment by FrannDzs** [comment]
> I think it is good. For future contributions to github projects, I'd suggest creating separate branches for each feature, instead of making the changes in the "master" branch directly.

Thank you for your help. I will take into account the branches, then I publish in 1.4, is that right?

**Comment by cesarBLG** [comment]
> > I think it is good. For future contributions to github projects, I'd suggest creating separate branches for each feature, instead of making the changes in the "master" branch directly.
> 
> Thank you for your help. I will take into account the branches, then I publish in 1.4, is that right?

What you have done is OK from our side, you have to compare against our "master" branch. My suggestion was because if you locally modify the master branch, upstream and local commit history will diverge. It is more convenient if you create a branch for each feature, based on the master branch.

I'm merging this now, so it's not necessary that you create a new branch for this.

**Pull request merged**